docs/source/index.rst

```diff
@@ -1,16 +1,25 @@
-.. title:: Goalie Goal-Oriented Mesh Adaptation Toolkit
+.. title:: Animate and Goalie
 
 .. only:: html
 
-Goalie Goal-Oriented Mesh Adaptation Toolkit
-============================================
+Welcome to the documentation for Animate and Goalie!
+====================================================
 
-Goalie provides metric-based goal-oriented mesh adaptation
-functionality to the Python-based finite element library
-`Firedrake <http://www.firedrakeproject.org/>`__. The 'y' is
-silent, so its pronunciation is identical to 'Proteus' - the
-ancient Greek god of the constantly changing surface of the
-sea.
+**Animate** is a package which provides anisotropic mesh adaptation
+functionality for the Python-based finite element library
+`Firedrake <http://www.firedrakeproject.org/>`__. Given an initial mesh, the
+user defines a Riemannian metric field (or simply 'metric'), which provides the
+'optimal mesh concept', i.e. a 'blueprint' for what adapted meshes will look
+like. Animate can then be used to transform the initial mesh into an adapted
+mesh, making use of the metric to guide the transformations.
+
+**Goalie** is more of an 'outer-loop' package, which is responsible for
+goal-oriented mesh adaptation pipelines for solving partial differential
+equations (PDEs) using adapted meshes. It supports solving time-dependent PDEs
```

Comment:
@jwallwork23 Is is worth mentioning here that Goalie executes a fixed point iteration loop?